shared/ui/core/src/main/java/com/ivy/ui/FormatMoneyUseCase.kt

```diff
@@ -8,6 +8,9 @@ import java.text.DecimalFormat
 import java.text.DecimalFormatSymbols
 import javax.inject.Inject
 
+const val MILLION = 1_000_000
+const val BILLION = 1_000_000_000
```

Comment:
We need to support also `k` for thousands

Reply:
Done